### ISSUE TITLE
Property chaining

### ISSUE THREAD

**Issue by rafek1241** [sandermvanvliet/SerilogSinksInMemory#15]
Actual
```
var a = ... .Appearing()
                .Once()
                .WithLevel(LogEventLevel.Error);
a 
                .WithProperty()
                .WithValue();

a 
                .WithProperty()
                .WithValue();
```

Expected:
```
a
                .WithProperty()
                .WithValue()
                .And
                .WithProperty()
                .WithValue();;
```

**Comment by hhko** [comment]
:cupid:

**Comment by sandermvanvliet** [comment]
@rafek1241 thanks for reporting this!

Sorry that it took a while to respond but this issue is fixed in #18 and will be available with version 0.7.0 of the assertions package on NuGet shortly.
